packages/manager/apps/ips/src/utils/getConfigValues.ts

```diff
@@ -0,0 +1,8 @@
+import { ProductConfiguration } from '@/data/hooks/catalog';
+
+export const getConfigValues = (
+  configs: ProductConfiguration[] = [],
+  configName: string,
+) => {
+  return configs.find((config) => config.name === configName)?.values || [];
+};
```

Comment:
I don't think it's a reusable utils outside of this context
You can keep the function close to where it is used instead